spec/features/events/new_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+module Features
+  describe 'fulfil order' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.